hack/boilerplate/boilerplate.py

```diff
@@ -202,9 +202,12 @@ def get_regexs():
     regexs["year"] = re.compile('YEAR')
     # dates can be 2014, 2015, 2016, or 2017; company holder names can be anything
     regexs["date"] = re.compile('(2014|2015|2016|2017)')
-    # strip // +build \n\n build constraints
-    regexs["go_build_constraints"] = re.compile(r"^(// \+build.*\n)+\n",
-                                                re.MULTILINE)
+    # strip the following build constraints/tags:
```

Comment:
comment: this is for new go 1.18 style go build tags.

thought: what do we think about this script in general? It's not 2017 any more!

Reply:
Yes, something to resolve but not for this PR.

Reply:
I def think we can do better!